mep3_robot/launch/robot_launch.py

```diff
@@ -96,6 +65,15 @@ def generate_launch_description():
         condition=launch.conditions.IfCondition(use_rviz)
     )
 
+    behavior_tree = Node(
+        package='mep3_planning',
+        executable='mep3_planning',
+        output='screen',
+        arguments=['ros_demo'],
```

Comment:
```suggestion
        arguments=['robot_strategy_1'],
```
Ovo ne radi, treba negdje u `mep3_planning` sacekati da se Nav2 paket podigne, slicno ovome:
https://github.com/memristor/mep3/blob/2f33eedd642c46e308bff5e5499999746578ef31/mep3_planning/include/mep3_planning/navigate_to_action.hpp#L42

Reply:
Ovo je relativno sredjeno u d5c2094

Generalno svakako necemo imati ovaj problem zato sto svejedno robot krece na povlacanje konopca (kada se sve komponente inicijalizuju)